app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,52 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.all
+  end
+
+  def new
+    @album = Album.new
+  end
+
+  def create
+    @album = Album.new(album_params)
+    if @album.save
+      redirect_to action: 'index'
+    else
+      render :new
+    end
+  end
+
+  def show
+    @album = get_album
+  end
+
+  def edit
+    @album = get_album
+  end
+
+  def update
+    @album = Album.find(params[:id])
+    if @album.update_attributes(album_params)
+      redirect_to action: 'index'
+    else
+      render :edit
+    end
+  end
+
+    def destroy
+      Album.find_by(id: params['id']).destroy
```

Comment:
This is because you've already got a method that performs this work and b/c you may want to find it, perform some logic on it (like can it be destroyed?), and then destroy it